modular_skyrat/modules/customization/modules/surgery/organs/ipc.dm

```diff
@@ -19,11 +19,11 @@
 /obj/item/organ/brain/ipc_positron/emp_act(severity)
 	switch(severity)
 		if(1)
-			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 75, 150)
-			to_chat(owner, span_warning("Alert: Posibrain heavily damaged."))
+			owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 36, 75)
```

Comment:
Those values should be defines, those severities should be defines too. I still don't know at a quick glance what a severity of one or a severity of two is.